src/plugins/files/server/blob_storage_service/adapters/es/content_stream/content_stream.ts

```diff
@@ -110,10 +139,22 @@ export class ContentStream extends Duplex {
         chunks.push(chunk);
       }
       const buffer = Buffer.concat(chunks);
-      const source: undefined | FileChunkDocument = buffer.byteLength
-        ? cborx.decode(buffer)?._source
+      const decodedChunkDoc: GetResponse<FileChunkDocument> | undefined = buffer.byteLength
+        ? (cborx.decode(buffer) as GetResponse<FileChunkDocument>)
         : undefined;
 
+      // Because `asStream` was used in retrieving the document, errors will also not be processes
```

Comment:
FYI: this was something that had caused me some trouble in the past. When `asStream` is used, errors in ES are also not processed, so the call does not actually fail. We check here to see if retrieving the chunk was successful and error/log it if not.